x/nft/types/id_token.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import (
+	"fmt"
+	"strconv"
+)
+
+const (
+	// token id must be longer thant 3 length
+	MIN_TOKEN_ID = uint64(1)
+	// iris nft module restrict only number denom id, therefore appending alphanumeric prefix
+	TOKEN_ID_PREFIX = "tokenid"
+)
+
+type TokenID uint64
```

Comment:
iris token id is string, instead nibiru token id is number in order to allocate id incrementaly.